dts/bindings/sdhc/emmc-host.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: eMMC host controller
+
+compatible: "emmc-host"
```

Comment:
this naming is too generic, it is confusing, it should at least be prefixed with the vendor name like `intel,emmc-host`

Reply:
the driver is compline to the standard host specification and shall work with the most of the EMMC host controller IP.  That is why I given it a generic eMMC Host compatibility string.

Reply:
What devices has this driver been tested on? If we can verify this driver functions on multiple host controller IP blocks, I am ok with a more generic binding name. I suspect we will need to remove (or conventionalize) the PCIE macros to do this though.

If we want to introduce this as a generic driver though, I would prefer we expose the SD host controller function implementations in a header file similar to how Linux does, or create an `sdhc_common.c` file that this driver (and others) can utilize). That way vendors can leverage the generic SDHC functions where possible in their driver implementations, and override the SDHC functions they need to.

Reply:
Currently I will change the compatibility id to "intel,emmc-host" since I tested only with Intel platform. But later someone can update as per your suggested once they tested with different platform.